UPGRADING.md

```diff
@@ -42,6 +42,6 @@ var db = level('DB')
 var stream = WriteStream(db, { type: 'del' })
 ```
 
-`stream.writable` and `stream.readable` states were removed so those properties can no longer be accessed.
+Internal `this.writable` and `this.readable` was removed. However, `this.writable` still exists due to inheritance, but `this.readable` is now `undefined`.
```

Comment:
Did you mean "internally"?

Reply:
As in, the properties that used to be defined in _this_ constructor were removed.